src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/controllers/LinkDefendantToClaimViaAuthenticationTest.java

```diff
@@ -42,10 +46,17 @@ public void shouldReturn200HttpStatusAndUpdatedClaimWhenLinkIsSuccessfullySet()
         );
 
         Claim claim = claimStore.saveClaim(SampleClaimData.builder().build());
-        String urlTemplate = String.format(urlFormat, claim.getReferenceNumber(), username, password);
+        String urlTemplate = String.format(urlFormat, claim.getReferenceNumber());
 
-        webClient
-            .perform(put(urlTemplate))
+        MultiValueMap<String, String> params = new LinkedMultiValueMap<>();
```

Comment:
should we use MultiValuedMap instead of MultiValueMap? - 

https://commons.apache.org/proper/commons-collections/apidocs/org/apache/commons/collections4/map/MultiValueMap.html

Just for my knowledge - do we need MultiValueMap here since we seem to have different values for different keys?

Reply:
The API for the params expects a MultiValueMap. My assumption here is it allows for passing arrays of values for a given key (though why they don't just use an array or collection object here is beyond me).